tests/queries/0_stateless/02981_vertical_merges_memory_usage.sql

```diff
@@ -14,7 +14,7 @@ SETTINGS
     merge_max_block_size_bytes = '10M';
 
 INSERT INTO t_vertical_merge_memory SELECT number, arrayMap(x -> repeat('a', 50), range(1000)) FROM numbers(3000);
-INSERT INTO t_vertical_merge_memory SELECT number, arrayMap(x -> repeat('a', 50), range(1000)) FROM numbers(3000);
+INSERT INTO t_vertical_merge_memory SELECT number, arrayMap(x -> repeat('a', 50), range(1000)) FROM numbers(3001);
```

Comment:
🤔

Reply:
Deduplication, which is off with normal MergeTree by default

Reply:
Maybe we should randomize `non_replicated_deduplication_window`, but in another PR.